masterlist.yaml

```diff
@@ -17632,12 +17910,15 @@ plugins:
       - 'URN+CoT Patch.esp'
       - 'Atlas Map Markers.esp'
       - 'The Horkerpocalypse.esp'
-    tag:
-      - C.Location
```

Comment:
It's been moved to below the `msg` block. I'd rather it wasn't, it's just noise, but I don't consider it a blocker.